benchmarks/full_catalog/hiera.yaml.erb

```diff
@@ -5,7 +5,6 @@
   :datadir: "<%= datadir %>"
 :hierarchy:
 # This hierarchy is based on the one from the puppetlabs ops team.
-  - "vagrant/%{is_vagrant}"
```

Comment:
Does `osfamily` (below) need to be updated too?

Reply:
I don't believe so, it didn't fail for me and I _think_ it's fine because osfamily is part of the fact fixtures here: https://github.com/puppetlabs/puppet/pull/9019/files#diff-1800058a90b803c89caa186ab07f38df7b2bd7db904cca12dc860a821f976d26L62

Is osfamily a fact that went away and should be removed from the fixtures?

Reply:
yeah it's a legacy fact replaced with the `os.family` structured fact. But since the fact is provided by the fixture then it wouldn't fail.

Reply:
After pulling on this thread for a bit, I'm going to say "No". The osfamily fact should be turned into a structured fact along with fqdn & hostname being nested within the networking structure. To do that requires updating a lot of modules and the calling code in our perf_control module. That should be done, but I've filed https://tickets.puppetlabs.com/browse/SERVER-3260 for us to do it post Puppet 8.